### DISCUSSION TITLE
Allowing users to upload documents

### DISCUSSION THREAD

**Discussion by wecr3at3** [FlowiseAI/Flowise#831] [Q&A]
Hello, 

So far I have been able to create a conversational QnA chatbot using my own data via document loaders but, I am working on this use case of an AI HR Assistant that allows users (HR personnel) to upload documents (docx, pdfs, txt) like CVs and job descriptions, Policy documents, etc.

How can I achieve this?

**Comment by chungyau97** [comment]
Hi @wecr3at3,

It's currently not in our top priorities and I'm not able to give you an estimate when will this be rolling out.

The current workaround:
Modify the source code in [Flowise Chat Embed](https://github.com/FlowiseAI/FlowiseChatEmbed) for the UI and the request body to allow override config.

References: 
1. [Flowise Chat Embed docs](https://docs.flowiseai.com/how-to-use#embed)
2. [Flowise API examples](https://docs.flowiseai.com/how-to-use#api)

**Comment by HenryHengZJ** [comment]
A full tutorial using API https://youtu.be/s33v5cIeqA4?si=6WL6wATyFHrUQDBs
